README.md

```diff
@@ -195,7 +195,26 @@ Make sure you are using an unmultiplied (straight) version of your color source:
 
 Note that the canvas API is subject to cross domain security restrictions, so be aware that the video source files have to be served from the same domain (i.e. if the document that is calling `seeThru` is on `www.example.net` the video files have to be requested from `www.example.net` as well), otherwise you will see a DOM Security Exception. Also note that this also applies to subdomains, therefore you cannot mix www and non-www-URLs.
 
-This can be worked around when using **[CORS][28]**.
+This can be worked around when using **[CORS][28]** or by using objectURL:
+
+````Javascript
+function loadAsObjectURL(video, url) {
+    var xhr = new XMLHttpRequest();
+    xhr.responseType = 'blob';
+    xhr.onload = function (response) { return video.src = URL.createObjectURL(xhr.response); };
+    xhr.onerror = function () { /* Huston we have a problem! */ };
```

Comment:
😳  Thanks - I've always misspelled that (why didn't anyone correct me until now)